### PR TITLE
Avatars and alerts

### PR DESCRIPTION
**Pull request by amantulsyan35**
1. avatar-component
2. alert-component
Reviews are highly appreciated :))

### CODE REVIEW

File: css/components/avatar.css [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+.avatar {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  border-radius: 50%;
+  position: relative;
+}
+
+/* .avatar-name {
```
Comment (anasm11) [inline css/components/avatar.css:11]: comments can be removed

File: css/components/alert.css [hunk 2]
```diff
@@ -0,0 +1,42 @@
+.alert {
+  display: flex;
+  flex-direction: row;
```
Comment (anasm11) [inline css/components/alert.css]: flex-direction is row by default

Comment (amantulsyan35) [reply css/components/alert.css]: Thanks will change that

**Comment by Naman-Saxena1** [comment]
Everything looks fine! :+1:

File: css/components/avatar.css (hunk 1 above)
Comment (amantulsyan35) [reply css/components/avatar.css:11]: need it later, want to add something

**Pull request merged**